app/assets/javascripts/test/geometries/skeleton.spec.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
no more flow?

Reply:
This is a test file which was previously not typechecked anyway, since the entire test folder was blacklisted. I changed it so that only the files which fail the typechecks currently are not checked by removing the flow-directive explicitly.